about.html

```diff
@@ -0,0 +1,125 @@
+            <div class="desktop-menu">
+                   <a id="logo" id="homeLink" href="./index.html"> <img src="style/images/logo.png" alt="logo"></a>
+                <ul>
+                     <li><a href="./about.html" id="aboutLink" id="active">About</a></li>
+                    <li><a href="#">Program</a> </li>
+                    <li><a href="#">Join</a> </li>
+                    <li><a href="#">Contact</a> </li>
+                    <li><a href="#" class="compaign">Compaign</a> </li>
+                </ul>
+            </div>
+        </nav>
+    </section>
+
+    <section class="headline2">
+        <div class="tittle">
+            <h3>"Hello! Coding Amateur"</h3>
+            <h1>SUMMER CRASH BOOTCAMP </h1>
+        </div>
+        
+        <div class="headline21">
+            <p class="headline-desc2">Our summer Bootcamp comes with new features, it brings together a community of experts, academics, and activists who comprise great tech corporations. This year, we hope to expand our student list from all over the world.</p>
+            <p class="p">Please contact us per Email for any further questions about summer Bootcamp!</p>
+            <p class="address">summer.bootcamp2023@morocco.org</p>
+        </div>
+
+        <div class="headline-desc21">
+            <h2>Summer BootCamp<br class="null"> Crash Courses 2023</h2>
+            <hr>
+            <p>The course kicks off on June 20 and ends on August 15, 2023.</p>
+            <img class="boot-image" src="style/images/logo.png" alt="">
+        </div>
+        <div class="past-camp">
+            <h3>See the past summer BootCamp</h3>
+            <hr>
+            <p>Take a look at the last two Summer BootCamp whic took place in Geneva and in Cape Town</p>
+
+            <div class="img-all">
+                <div class="image-container1 image-content">
+                        <h3>2022</h3>
+                        <p>Our BootCamp in Geneva</p>
+                </div>
+                <div class="image-container2 image-content">
+                        <h3>2021</h3>
+                        <p>Our BootCamp in Cape Town</p>
+                </div> 
+            </div>
+        </div> 
+    </section>
+
+
+    <footer>
+        <div class="partner">
+                <h2>Partner</h2>
+                <hr>
+
+            <div class="partner-corporation">
+                    <h3><i class="fab fa-microsoft"></i>Microsoft</h3>
+                    <h3><i class="fab fa-google"></i>Google</h3>
+                    <h3><i class="fa-brands fa-facebook"></i>FaceBook</h3>
+                    <h3 class="item"><i class="fab fa-apple"></i>Apple</h3>
+                    <h3 class="item"><i class="fa-brands fa-airbnb"></i>airbnb</h3>
+            </div>  
+        </div>
+
+        <div class="logo-rights">
+            <img src="style/images/logo.png" alt="creative-common">
+            <p>@2023 summer BootCamp Some rights reserved</p>
+        </div>
```

Comment:
- [ ] On the desktop view the logo is very small and the text is too large and it is too tall. Kindly increase the size of the logo and decrease the size of the text and decrease the height so your footer can look more similar to the template and look more professional. 
 - Your design:
 - 
<img width="964" alt="image" src="https://user-images.githubusercontent.com/61919617/203774168-6cce7911-0a97-41ee-b898-cf9289f888c0.png">

 - Template:
 - 
<img width="762" alt="image" src="https://user-images.githubusercontent.com/61919617/203774273-af3b7ff9-c44e-4ea0-8ebc-92c4a77db42d.png">